lib/matplotlib/tests/test_axes.py

```diff
@@ -8545,3 +8545,9 @@ def test_ecdf_invalid():
         plt.ecdf([1, np.nan])
     with pytest.raises(ValueError):
         plt.ecdf(np.ma.array([1, 2], mask=[True, False]))
+
+
+def test_errorbar_mapview_kwarg():
+    D = {ii: ii for ii in range(10)}
+    fig, ax = plt.subplots()
+    plt.errorbar(D.keys(), D.values(), D.values())
```

Comment:
```suggestion
    ax.errorbar(x=D.keys(), y=D.values(), xerr=D.values())
```

Slight preference for this presentation, as it is a bit more explicit about the `kwarg` nature of the change to `_preprocess_data` (pyplot passes _everything_ as kwargs, so it still triggered the problem, but this is just a bit more clear)

Reply:
Good point, added the change